### DISCUSSION TITLE
Corrupted zip?

### DISCUSSION THREAD

**Discussion by vulturo26** [rickloader/NewForestRouteV3#12] [Q&A]
After downloading the New Forest zip version I got a 'This file is corrupted' message when I tried to unpack it (with WinRAR). I deleted the first download and tried again - same result. Any thoughts?

**Comment by rickloader** [comment]
You could try 7-zip. I don`t recommend the .zip download. I believe the zip download can time out unless you have fast internet.
It is far better to install Github Desktop, and has the bonus that you get easy updates.
Thanks for your interest in NF Route
Rick
